Azeem/src/DLMFtex2Wiki.py

```diff
@@ -3,181 +3,22 @@
-    length=0
-    for c in line:
-        if fEq and c!=" " and c!="$" and c!="{" and c!="}" and not c.isalpha():
-            length+=1
-        if count>=0 and per!=0:
-            per+=1
-            if c!=" " and c!="%":
-            #if c!="%":
-                per=0
-        if c=="{" and per==0:
-            if count>0:
-                stringWrite+=c
-            count+=1
-        elif c=="}" and per==0:
-            count-=1
-            if count>0:
-                stringWrite+=c
-        elif c=="$" and per==0:
-            fEq=not(fEq)
-            if fEq:
-                stringWrite+="<math>{\\displaystyle "
-            else:
-                if length<10:
-                    stringWrite+="}</math>"
-                else:
-                    stringWrite+="}</math>"+"<br />"
-                length=0
-        elif c=="\n" and per==0:
-            stringWrite=stringWrite.strip()
-            stringWrite+=c
-            per+=1
-        elif count>0 and per==0:
-            stringWrite+=c
-
-    return (stringWrite.lstrip())
-def getSym(line): #Gets all symbols on a line for symbols list
-    symList=[]
-    if line=="":
-        return symList
-    symbol=""
-    symFlag=False
-    cC=0
-    for i in range(0,len(line)):
-        #if "MeixnerPollaczek{\\lambda}{n}@{x}{\\phi}=\frac{\\pochhammer{2\\lambda}{n}}" in line:
-        c=line[i]
-        if symFlag:
-            if c=="{" or c=="[":
-                cC+=1
-            if c!="}" and c!="]":
-                if c.isalpha():
-                    symbol+=c
-                else:
-                    symFlag=False
-                    symList.append(symbol)
-                    symList+=(getSym(symbol))
-                    symbol=""
-            else:
-                cC-=1
-                symbol+=c
-                if i+1==len(line):
-                    p=""
-                else:
-                    p=line[i+1]
-                if cC<=0 and p!="{" and p!="[" and p!="@":
-                    symFlag=False
-                    #if "monicAskeyWilson{n+1}@{x}{a}{b}{c}{d}{q}+\\frac{1}{2}" in line:
-                    symList.append(symbol)
-                    symList+=(getSym(symbol))
-                    symbol=""
-
-        elif c=="\\":
-            symFlag=True
-        elif c=="&" and not (i>line.find("\\begin{array}") and i<line.find("\\end{array}")):
-            symList.append("&")
-
-    #if "MeixnerPollaczek{\\lambda}{n}@{x}{\\phi}=\frac{\\pochhammer{2\\lambda}{n}}" in line:
-    symList.append(symbol)
-    symList+=getSym(symbol)
-    return(symList)
-def unmodLabel(label):
-    label=label.replace(".0",".")
-    label=label.replace(":0",":")
-    return(label)
-def secLabel(label):
-    return(label.replace("\'\'",""))
+
 def modLabel(label):
     #label.replace("Formula:KLS:","KLS;")
     isNumer=False
     newlabel=""
     num=""
     for i in range(0,len(label)):
-        if isNumer and not isnumber(label[i]):
+        if isNumer and not is_number(label[i]):
```

Comment:
As discussed with @pwang00 this fixes a bug introduced before and thus is the intended behaviour.